<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlTabWidget</class>
 <widget class="QWidget" name="ControlTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidgetControl">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="BannaWidget" name="tabBanna">
      <attribute name="title">
       <string>轮播图</string>
      </attribute>
     </widget>
     <widget class="CylinderWidget" name="tabCylinder">
      <attribute name="title">
       <string> 柱状图</string>
      </attribute>
     </widget>
     <widget class="ProgressWidget" name="tabProgress">
      <attribute name="title">
       <string>进度条</string>
      </attribute>
     </widget>
     <widget class="FrameWidget" name="tabFrame">
      <attribute name="title">
       <string>面板</string>
      </attribute>
     </widget>
     <widget class="PlotWidget" name="tabPlot">
      <attribute name="title">
       <string> 坐标系</string>
      </attribute>
     </widget>
     <widget class="MoveWidget" name="tabMoveButton">
      <attribute name="title">
       <string>可移动按钮</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BannaWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\bannacontrol\bannawidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CylinderWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\cylindercontrol\cylinderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\progresscontrol\progresswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\framecontrol\framewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\plotcontrol\plotwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MoveWidget</class>
   <extends>QWidget</extends>
   <header location="global">maincontent\controlwidget\movecontrol\movewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidgetControl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
